CHANGELOG.md

```diff
@@ -10,9 +10,13 @@
 
 <!-- next-header -->
 
-## [[UnreleasedUniFFIVersion]] (backend crates: [[UnreleasedBackendVersion]] - (_[[ReleaseDate]]_)
+## [[UnreleasedUniFFIVersion]] (backend crates: [[UnreleasedBackendVersion]]) - (_[[ReleaseDate]]_)
 
-[All changes in [[UnreleasedVersion]]](https://github.com/mozilla/uniffi-rs/compare/v0.22.0...HEAD).
+[All changes in [[UnreleasedVersion]]](https://github.com/mozilla/uniffi-rs/compare/v0.23.0...HEAD).
+
+## v0.23.0 (backend crates: v0.23.0) - (_2023-01-27_)
+
+[All changes in v0.23.0](https://github.com/mozilla/uniffi-rs/compare/v0.22.0...{{tag_name}}).
```

Comment:
This should be:

```suggestion
[All changes in v0.23.0](https://github.com/mozilla/uniffi-rs/compare/v0.22.0...v0.23.0).
```

I assume this is handled by `cargo release`? Is this an issue with our defined replacements or a cargo-release issue?